node/src/components/consensus/era_supervisor/era.rs

```diff
@@ -49,6 +49,10 @@ impl EraId {
     pub(crate) fn checked_sub(&self, x: u64) -> Option<EraId> {
         self.0.checked_sub(x).map(EraId)
     }
+
+    pub(crate) fn value(&self) -> u64 {
+        self.0
+    }
```

Comment:
Maybe `impl Into<u64> for EraId` would be more appropriate. We could even derive that with `derive_more`.

Reply:
Did this @afck